.github/workflows/ci.yml

```diff
@@ -78,7 +78,6 @@ jobs:
       - aws-auth
       - build
     uses: ./.github/workflows/terraform-plan.yml
-    if: needs.build.outputs.build-api-image-result == 'success' && needs.build.outputs.build-web-result == 'success' && needs.aws-auth.result == 'success'
```

Comment:
Alternate fix:
```yaml
if: needs.build.outputs.build-console-image-result == 'success' && needs.build.outputs.build-web-result == 'success' && needs.aws-auth.result == 'success'
```

As mentioned in the opening comment, I don't believe this conditional is doing anything useful, which is why I removed it altogether. The above line is also suitable, as it gates the conditional on a successful Docker build of the RedwoodJS console image. 

API image builds are currently skipped since they were taking a while to run and aren't actually being deployed anywhere, which is why this check was always evaluating to `false`.